internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -385,6 +386,9 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) openAIMessageToBedrockMes
 func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) ResponseHeaders(headers map[string]string) (
 	headerMutation *extprocv3.HeaderMutation, err error,
 ) {
+	if v, ok := headers[statusHeaderName]; ok && v != string(typev3.StatusCode_OK) {
+		return nil, nil
+	}
```

Comment:
I think this is not necessary or should be panic... if we reach this, that means a bug in extproc code